lib/landable/traffic/user_tracker.rb

```diff
@@ -38,8 +38,17 @@ def record_page_view
         end
       end
 
+      def amqp_config_hash
+        @amqp_config_hash = Landable.configuration.amqp_configuration
```

Comment:
again i dont think you need to state ```config_hash`` so I would just make this ```@amqp = Landable.configuration.amqp_configuration```